README.md

```diff
@@ -42,22 +42,92 @@ A visual studio code build task is also included so you can run it from there wi
 ## Running the CLI
 Install the CLI with either npm install -g .  or npm install -g dicomp10-to-dicomweb
 
+### Api
+There are 5 different commands: mkdicomweb, mkdicomwebinstances, mkdicomwebstudy, mkdicomwebdeduplicated, mkdicomwebdeduplicatedgroup that can be used (see usage bellow). It can be run as:
+
+```
+commnadName [options]
+```
+The options are:
+
+  -c, --clean
+  : Clean the study output directory for these instances (default: false)
+  
+  -C, --remove-deduplicated-instances
+  : Remove single instance deduplicated files after writing group files (default: false)
+  
+  -d, --deduplicate
+  : Write deduplicate instance level data (default: false)
+  
+  -g, --group 
+  : Write combined deduplicate data (default: true)
+  
+  -h, --help
+  : Print help (default: false)
+  
+  -i, --input <input...>
+  : List of files/directories/studyUids to be processed (default: [])
+  Required for commands: **mkdicomweb**, **mkdicomwebdeduplicated**, **mkdicomwebinstances**
+  
+  -I, --instances
+  : Write instance metadata (default: false)
+  
+  -m, --maximum-inline-public-length <value>
+  : Maximum length of public binary data (default: 131074)
+  
+  -M, --maximum-inline-private-length <value>
+  : Maximum length of private binary data (default: 64)
+  
+  --no-recompress
+  : Force no recompression
+  [see](#to-recompress)
+  
+  -o, --dir <value>
+  : Set output directory (default: "~/dicomweb")
+  
+  --path-deduplicated <path>
+  : Set the deduplicate data directory path (relative to dir) (default: "deduplicated")
+  
+  --path-instances <path>
+  : Set the instances directory path (relative to dir) (default: "instances")
+  
+  -r, --recompress <listvalue...> 
+  : List of types to recompress separated by comma (choices: "uncompressed", "jp2", "jpeglossless", "rle", default: "uncompressed jp2")
+  [see](#to-recompress)
+  
+  -s, --study
+  : Write study metadata - on provided instances only (TO FIX}, (default: true)
+  
+  -t, --content-type <type>
+  : Content type (default: null)
+  
+  -T, --colour-content-type <value>
+  : Colour content type (default: null)
+  
+  -v, --verbose
+  : Write verbose output (default: false)
+  
+  -V, --version
+  : output the version number
+
+
+
 ### To create instances
 Run the tool:
 ```
-mkdicomwebinstances <directoryOfP10Files>
+mkdicomwebinstances -i <directoryOfP10Files>
```

Comment:
Please fix this so that the remaining arguments are the extra path files - this should NOT require a -i here as one wants to be able to list multiple incoming file names.